bridge/util.go

```diff
@@ -80,6 +80,11 @@ func serviceMetaData(config *dockerapi.Config, port string) (map[string]string,
 				metadata[key] = kvp[1]
 			}
 		}
+		if (kvp[0] == "com.amazonaws.ecs.task-arn") && len(kvp) > 1 {
+			key := "taskId"
+			value := strings.SplitN(value, "/", -1)[2]
```

Comment:
You probably want to split `kvp[1]` yeah?

```suggestion
			value := strings.SplitN(kvp[1], "/", 3)[2]
```

Reply:
Good call :facepalm: